### ISSUE TITLE
Build overwrite existing packages

### ISSUE THREAD

**Issue by Hakuhun** [davidmc24/gradle-avro-plugin#94]
Hi! 

If a project already contains packages of the generated classes and the output destination set where the those packages are, the class generation deletes all the existing packages.

I use a custome gradle task to prevent that, but I think it'd be great if the not-overwriting version could be implemented.

```
task deleteOriginal(type: Delete){
    delete "src/main/avro"
}

task copySource(type: Copy) {
    from "src/main/avro"
    into "src/main/java"
}

task generateAvro(type: com.commercehub.gradle.plugin.avro.GenerateAvroJavaTask) {
    source("src/main/resources/avro")
    outputDir = file("src/main/avro")
    generateAvro.finalizedBy copySource
    generateAvro.finalizedBy deleteOriginal
}
```

**Comment by davidmc24** [comment]
I believe that the pre-clean behavior you're talking about was already removed in version 0.13.0.  Are you still seeing this behavior in version 0.17.0?

**Comment by davidmc24** [comment]
Possible duplicate of #51

**Comment by Hakuhun** [comment]
> I believe that the pre-clean behavior you're talking about was already removed in version 0.13.0. Are you still seeing this behavior in version 0.17.0?

Updating the version solved the error. The dependecy was set to 0.10.0. You should update the README. 
You would save the life of the copypasters like me. :D

**Comment by davidmc24** [comment]
@Hakuhun Thank you for the suggestion to improve the documentation.

I have checked, and the feature change is already noted in the changelog (both in https://github.com/davidmc24/gradle-avro-plugin/releases and https://github.com/davidmc24/gradle-avro-plugin/blob/master/CHANGES.md).

I've added issue templates for both bug reports and feature requests that clearly call out the need to check if you're on the latest version.

The README is intended to provide guidance on the current latest version; as such, I won't be adding a call-out for a behavior that hasn't been present in the latest version for almost 2 years.
